_docker/docker-compose.yml.erb

```diff
@@ -52,6 +83,10 @@ awestruct_preview_no_reload:
     - DRUPAL_HOST_PORT=<%=ENV['DRUPAL_HOST_PORT']%>
     - DRUPAL_HOST_IP
     - HOST_TO_TEST
+    - SELENIUM_HOST
```

Comment:
- HOST_TO_TEST
     - SELENIUM_HOST
I don't see why these are needed. awestruct_preview_no_reload is not responsible for testing